libraries/adaptive-expressions/src/builtinFunctions/empty.ts

```diff
@@ -36,16 +35,14 @@ export class Empty extends ComparisonEvaluator {
      * @private
      */
     private static isEmpty(instance: any): boolean {
-        let result: boolean;
-        if (instance === undefined) {
-            result = true;
-        } else if (typeof instance === 'string') {
+        let result = false;
```

Comment:
`isEmpty(undefined)` returns `false` here - is that the intended behavior?

Reply:
There is indeed a small bug for empty. I create an independent issue here: https://github.com/microsoft/botbuilder-dotnet/issues/5337. I would drop the chage of "empty" for this pr, and would port it after C# side is done. Thank.